packages/react/src/components/Storage/FileUploader/displayText.ts

```diff
@@ -0,0 +1,35 @@
+export const defaultDisplayText = {
+  filesUploaded(count: number): string {
+    return `${count} ${count === 1 ? 'file uploaded' : 'files uploaded'}`;
+  },
+  remainingFiles(count: number): string {
+    return `${count} ${count === 1 ? 'file' : 'files'} selected`;
+  },
+  uploading(percentage: number): string {
+    return `Uploading${percentage > 0 ? `: ${percentage}%` : ''}`;
+  },
+  uploadButton(count: number): string {
+    return `Upload ${count} ${count === 1 ? 'file' : 'files'}`;
+  },
+  maxFilesError(count: number): string {
+    return `Cannot choose more than ${count} ${
+      count === 1 ? 'file' : 'files'
+    }. Remove files before updating`;
+  },
+  error(message: string): string {
+    return message;
+  },
+  doneButton: 'Done',
+  clearButton: 'Clear all',
+  extensionNotAllowed: 'Extension not allowed',
```

Comment:
* The unit tests should also be updated to use `defaultDisplayText` instead of hardcoded values.
* I noticed we are not passing these strings through the translation module anymore. Is that intentional? Wondering if it could be a breaking change for customers who added their own translations.

Reply:
Correct, and yes. When I add the docs I will add a section on using the Amplify.i18n module